src/editor/init.js

```diff
@@ -7,15 +7,8 @@
 var _initialized = false;
 var _defaultTheme = "";
 
-function addStylesheet(path) {
-    var link = document.createElement("link");
-    link.href = path;
-    link.type = "text/css";
-    link.rel = "stylesheet";
-    link.media = "screen,print";
-
-    document.getElementsByTagName("head")[0].appendChild(link);
-    return link;
+function addStylesheet(sheet){
+	document.getElementById('pagestyle').setAttribute('href', sheet);
```

Comment:
Yea... swapStylesheet is more appropriate... just haven't got around to name-changing, lol.